core/src/main/scala/org/broadinstitute/dsp/workbench/welder/Crc32c.scala

```diff
@@ -0,0 +1,117 @@
+package org.broadinstitute.dsp.workbench.welder
+
+import java.util.zip.Checksum
+
+import cats.effect.{ContextShift, IO}
+import com.google.common.io.BaseEncoding
+import org.broadinstitute.dsde.workbench.google2.Crc32
+import scala.concurrent.ExecutionContext.global
+// Adapted from https://github.com/googlearchive/crc32c-java/blob/master/src/com/google/cloud/Crc32c.java
```

Comment:
Is there a reason we can't use this directly? I'd be pretty surprised if there weren't an existing library to produce a crc32 in Scala/Java.

Reply:
This is the only java library I found(referenced [this page](https://cloud.google.com/storage/docs/hashes-etags), and it's not published. I've tested the hash generated vs gsutil, so it should be correct unless google changes the algorithm

Reply:
What about https://docs.oracle.com/javase/8/docs/api/java/util/zip/CRC32.html?

Reply:
Google's doc says their crc32 is different from the regular java one

Reply:
Oh missed the `c` :) ok

Reply:
I also missed this initially - that's what threw me off. This makes sense then. I would probably add a comment somewhere in this file that mentions this is a variation of CRC32 to help future readers.

As an alternative, we could probably use MD5 instead if this presents any problems (but I think current approach is fine AFAICT for now).

Reply:
It's worth also making it clear that it's **crc32c** as that makes it easier to dig up information